chore: add different templates for issue (#1853)
DevExpress/devextreme-reactive: .github/ISSUE_TEMPLATE/support_question.md

<!--
    We cannot guarantee a quick answer on GitHub.
    If you own an active DevExtreme license, feel free to contact us in the Support Center (https://www.devexpress.com/ask)
    to receive an answer shortly.
-->

<!--
    A checked check box should look like this: [x]
-->
- [ ] I have searched this repository's [issues](https://github.com/devexpress/devextreme-reactive/issues) and believe that this is not a duplicate.

### I'm using ...

- [ ] React Grid
- [ ] React Chart
- [ ] React Scheduler

### Description
<!--
    Ask your question. Provide as many details as possible.
-->

### Environment
<!---

>     Include as many relevant details about the environment with which you use our components. Leave "none" if you don't have the package in your package.json file.

Reviewer: ```suggestion
    Include as many relevant details about the environment you use. Leave "none" if you do not have the package in your package.json file.
```
